fe/fe-core/src/main/java/com/starrocks/transaction/PublishVersionDaemon.java

```diff
@@ -379,6 +381,7 @@ private boolean publishPartition(@NotNull Database db, @NotNull TableCommitInfo
         long tableId = tableCommitInfo.getTableId();
         long txnVersion = partitionCommitInfo.getVersion();
         long txnId = txnState.getTransactionId();
+        long commitTime = txnState.getCommitTime();
```

Comment:
If publish version is slow, commitTime may be a long time ago. why not use the current time?

Reply:
current time is not a deterministic value, violating the idempotency constraints